tests/parser/functions/test_send.py

```diff
@@ -56,3 +56,47 @@ def __default__():
     assert receiver.last_sender() == sender.address
     assert w3.eth.get_balance(sender.address) == 0
     assert w3.eth.get_balance(receiver.address) == 1
+
+
+def test_gas_kwarg(get_contract, w3):
+    """
+    Tests to verify that send to default function will send limited gas (2300),
+    but raw_call can send more.
+    """
+
+    sender_code = """
+@external
+def test_send(receiver: address):
+    send(receiver, 1, gas=50000)
+
+@external
+def test_call(receiver: address):
+    raw_call(receiver, b"", gas=50000, max_outsize=0, value=1)
```

Comment:
not sure raw_call needs to be tested here

Reply:
gotcha